hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java

```diff
@@ -1704,7 +1703,7 @@ public void testScmRegisterNodeWithIpAddress()
   @Test
   public void testScmRegisterNodeWithHostname()
       throws IOException, InterruptedException, AuthenticationException {
-    testScmRegisterNodeWithNetworkTopology(true);
+    testScmRegisterNodeWithNetworkTopology();
   }
```

Comment:
These tests used to exercise the same use case with different config setting (use hostname true/false).  Since the config is being removed, they are now duplicates.  Calling the same underlying method from two different test methods without any difference is unnecessary.  Please remove the methods `...WithIpAddress` and `...WithHostname`, move the `@Test` annotation to the delegate methods, and remove the `private` modifier from those methods.